### PR TITLE
School Library Graphical User Interface

### PR DESCRIPTION
**Pull request by kennankole**
* Implemented the school library graphical user interface

### CODE REVIEW

File: src/menu.rb [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+      all_people
+    when 3
+      create_person
+    when 4
+      create_book
+    when 5
+      create_rental
+    when 6
+      list_all_rentals_by_id
+    else
```
Comment (Kamba56) [inline src/menu.rb]: - [ ] I can see you used the menu method to replace the main method suggested in the project requirements. As demanded by project requirements, please consider moving this to your menu method.

File: src/menu.rb [hunk 2]
```diff
@@ -0,0 +1,175 @@
+require './book'
+require './rental'
+require './student'
+require './teacher'
+require './persons'
```
Comment (Kamba56) [inline src/menu.rb]: - [ ] This is flagging some error on my end. A quick fix for the error is to change `require './file'` to `require_relative 'file'`. DO this in all your files and the error will be solved.
image

File: src/menu.rb [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+    if @people_list.empty?
+      puts 'There are not people yet'
+    else
+      @people_list.each do |person|
+        puts "Name: #{person.name}\n Age: #{person.age}"
+      end
+    end
+  end
+
+  def create_student
```
Comment (Kamba56) [inline src/menu.rb:72]: - [ ] I tried creating a student and a teacher and I am getting an error, kindly fix that please.
image

File: src/menu.rb [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+    puts 'Name too long' unless name.length.positive? && name.length < 100
+
+    puts "Specialization: \n"
+    specialiation = gets.chomp
+    specialiation = specialiation.downcase
+    @people_list << Teacher.new(id, age, name, specialiation)
+    puts 'Teacher created successfully'
+  end
+
+  def create_book
```
Comment (Kamba56) [inline src/menu.rb:105]: - [ ] I tried creating a book and I am getting an error kindly fix that please.

image

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @kennankole, 

Good job so far!!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [x] Great use of Github workflow!👏👏👏
- [x] All linters passed

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmiiorji — APPROVED** [review]
Hi @kennankole, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<div align="center">
  congratulations

</div>

## Good Highlights✔️

- App basic UI works
- Linter checks are passed
- Proper gitflow is followed


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**